internal/pkg/platform/schema/compiler/extension/extradata/mixin.go

```diff
@@ -0,0 +1,21 @@
+// Package extradata provide the Mixin for various additional entity fields.
+package extradata
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+	"github.com/keboola/go-utils/pkg/orderedmap"
+)
+
+type Mixin struct {
+	ent.Schema
+}
+
+func (v *Mixin) Fields() []ent.Field {
+	return []ent.Field{
+		field.
+			JSON("extra", &orderedmap.OrderedMap{}).
+			Comment("Extra data contains all entity fields that do not match the entity schema on parsing, used for forward compatibility").
+			Optional(),
+	}
+}
```

Comment:
Pri niektorych entitach, napr `Orchestration`, rozumieme iba casti definicie.
Relane je o konfiguracia, ktora ma aj nejake fields naviac + nejake fields sa mozu bez varovania pridat.

Aby s tym bolo mozne pracovat v CLI, tak musime tieto extra keys ulozit bokom.

Pri entitach, kde to chceme sa pouzije tento mixin, zatial nie je pouzity nikde.